client/app/scripts/superdesk-archive/module.js

```diff
@@ -140,8 +140,8 @@ define([
          * @param {Object} desk when passed the item will be assigned to this desk instead of user's activeDesk.
          */
         this.addTaskToArticle = function (item, desk) {
-            if ((!item.task || !item.task.desk) && (desk || desks.activeDeskId && desks.userDesks)) {
-                var currentDesk = desk || _.find(desks.userDesks._items, {_id: desks.activeDeskId});
+            if ((!item.task || !item.task.desk) && (desk || desks.getCurrentDeskId() && desks.userDesks)) {
```

Comment:
Agreed that there's no option to select personal in desks dropdown. My point was when user is either in custom workspaces or selects personal option from left then the item shouldn't get created in a desk and this is not been taken care in the changes.